QualityControl/public/layout/Layout.js

```diff
@@ -356,8 +356,8 @@ export default class Layout extends Observable {
    * @returns {undefined}
    */
   sortObjectsOfCurrentTab() {
-    this.gridList.items = this.tab.objects;
     if (this.editEnabled) {
```

Comment:
I believe this method is called only in EDIT mode already except for the call on line 394. Perhaps instead of this if condition, we should remove that call as well. 
Rest of the calls are in situations in which edit is enabled such as:
- adding object
- removing object
- resizing object